tartufo/cli.py

```diff
@@ -252,6 +252,13 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> Optional[click.Comma
     help="""Enable or disable terminal color. If not provided (default), enabled if
     output is a terminal (TTY).""",
 )
+@click.option(
+    "--no-local-config",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Skipping auto discovery and use of local config file",
+)
```

Comment:
This would look more like the other options if you structured this as `--local-config/--no-local-config` with the default being `True` (i.e. `--local-config`) -- look at the immediately following quiet definition for an example.